admin-ui/src/component/my-units/MyUnitRecurringReservation/MyUnitRecurringReservationForm.tsx

```diff
@@ -210,8 +210,8 @@ const MyUnitRecurringReservationForm = ({ reservationUnits }: Props) => {
   const checkedReservations = useFilteredReservationList({
     items: newReservations.reservations,
     reservationUnitPk: reservationUnit?.pk ?? undefined,
-    begin: getValues("startingDate"),
-    end: getValues("endingDate"),
+    begin: parse(getValues("startingDate"), "dd.MM.yyyy", new Date()),
+    end: parse(getValues("endingDate"), "dd.MM.yyyy", new Date()),
```

Comment:
There seems to be a lot of repetition in parsing and formatting the dates. A general helper function could ease readability.